File: CS/WindowsApplication1/RibbonForm1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;

namespace WindowsApplication1
{
    public partial class RibbonForm1 : DevExpress.XtraBars.Ribbon.RibbonForm
    {
        public RibbonForm1()
        {
            InitializeComponent();
        }

        private void barButtonItem1_ItemClick(object sender, ItemClickEventArgs e)
        {
            barEditItem1.Links[0].Focus();
            (barEditItem1.Links[0] as BarEditItemLink).ShowEditor();
        }

        private void barButtonItem2_ItemClick(object sender, ItemClickEventArgs e)
        {
            barEditItem2.Links[0].Focus();
            (barEditItem2.Links[0] as BarEditItemLink).ShowEditor();
        }
    }
}

File: VB/WindowsApplication1/RibbonForm1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars

Namespace WindowsApplication1
	Partial Public Class RibbonForm1
		Inherits DevExpress.XtraBars.Ribbon.RibbonForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub barButtonItem1_ItemClick(ByVal sender As Object, ByVal e As ItemClickEventArgs) Handles barButtonItem1.ItemClick
			barEditItem1.Links(0).Focus()
			TryCast(barEditItem1.Links(0), BarEditItemLink).ShowEditor()
		End Sub

		Private Sub barButtonItem2_ItemClick(ByVal sender As Object, ByVal e As ItemClickEventArgs) Handles barButtonItem2.ItemClick
			barEditItem2.Links(0).Focus()
			TryCast(barEditItem2.Links(0), BarEditItemLink).ShowEditor()
		End Sub
	End Class
End Namespace